scripts/tools/eslint-plugin-prettier-internal-rules/jsx-identifier-case.js

```diff
@@ -38,8 +42,4 @@ module.exports = {
       },
     };
   },
-  schema: {
-    type: "array",
-    uniqueItems: true,
-  },
```

Comment:
For ESLint rule, I'm not sure why schema `exists` here. Any reasons?

Reply:
My mistake.